content-style-guide.md

```diff
@@ -0,0 +1,87 @@
+# Salt content style guide
+
+Content written for Salt should follow this style guide. This includes component documentation, pattern guides and other website pages.
+
+Use American English and follow general guidance for producing clear content—concise, active voice, second person, present tense.
+
+## Guidance
+
+Entries are listed in alphabetical order.
+
+### Arrow keys
+
+The correct terms for the four arrow buttons on a keyboard are:
+ - <kbd>Up arrow</kbd>
+ - <kbd>Down arrow</kbd>
+ - <kbd>Left arrow</kbd>
+ - <kbd>Right arrow</kbd>
+
+Use \<kbd> tags where appropriate.
+
+### Best practices
+
+Use as a section header where necessary to draw the reader's attention to recommended tips and guidance. Never skip levels when using section headers.
```

Comment:
Is this referring to using 'Best practices' as a section header? Could that be clearer as I feel it is only implied, and I only understand it because I've seen a lot of our docs.

Also there is no reference to the way 'best practices' is used in the pattern docs (in the container highlighted green with the tick)